context/build-utils/webpack.common.js

```diff
@@ -78,7 +77,6 @@ const config = {
     ],
   },
   plugins: [
-    new CleanWebpackPlugin(),
```

Comment:
Is this going away because `mockServiceWorker.js` now lives in public, and we don't want to actually clear it out? So it's there in the production build, but nothing references it in that context?